### PR TITLE
fix: Webpack manifest plugin.

### PR DESCRIPTION
**Pull request by rajzik**
Fix void tag which throws an error when setting to true.
```js
Uncaught SyntaxError: Unexpected token '<'
```

### CODE REVIEW

**Comment by rajzik** [comment]
I know that this is a project for airbnb, but this was bug that I needed to fix as well.

File: packages/config-webpack/src/plugins/InlineManifestPlugin.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
-/* eslint-disable no-param-reassign, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access, import/no-extraneous-dependencies */
+/* eslint-disable no-param-reassign, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access */
```
Comment (milesj) [inline packages/config-webpack/src/plugins/InlineManifestPlugin.ts:1]: Why did you change the lints?

Comment (rajzik) [reply packages/config-webpack/src/plugins/InlineManifestPlugin.ts:1]: Eslint reported `Unused eslint-disable directive (no problems were reported from 'import/no-extraneous-dependencies').` 

But I will probably revert it.

**Comment by milesj** [comment]
We don't allow forks but I'll merge through since the change is real minor. Thanks!
